rasa/core/interpreter.py

```diff
@@ -296,6 +300,96 @@ def _load_interpreter(self) -> None:
 
         self.interpreter = Interpreter.load(self.model_directory)
 
+MODEL_NAME_PREFIX = "nlu"
+
+class RasaCoreInterpreter(NaturalLanguageInterpreter):
+    def __init__(self,):
+        self._load_interpreter()
+
+    def parse(self, text: Text):
+        result = self.interpreter.parse(text, only_output_properties=False)
+        return result
+
+    def _load_interpreter(self) -> None:
+        from rasa.nlu import config
+
+        config = config.load()
```

Comment:
we should load config from `config.yml`

Reply:
We load it from `config.yml` , just reusing the implemented NLU method for it

Reply:
but in `rasa.nlu.train` we actually load config from file: https://github.com/RasaHQ/rasa/blob/f87cc97c202a70ccb3004ff036112fa777353796/rasa/nlu/train.py#L69